### DISCUSSION TITLE
Can't configure new sensor - what am I doing wrong?

### DISCUSSION THREAD

**Discussion by caladrien** [mampfes/hacs_waste_collection_schedule#3010] [Q&A]
I'm new at this integration but since I realized, that I do not have to fill my calendar manually anymore, I'd like to use this integration.

So fare it is strait forward. Calendar already working for my. Filtered out the relevant collections:
 
![image](https://github.com/user-attachments/assets/dc4c53ea-c9b5-4590-99ea-995d847d1004)

What I now wanna have is one sensor for each type off collection showing me the remaining days till the collection. I thing the UI config is pretty straigt forward but somehow I make a mistake and i don't know where.

![image](https://github.com/user-attachments/assets/174dc875-625e-46ba-9136-744fad69d699)

![image](https://github.com/user-attachments/assets/50f9a17d-91cd-41da-a87f-9647a99b8eeb)

![image](https://github.com/user-attachments/assets/eecb367b-a8f7-4883-b220-aff551d0ac89)
![image](https://github.com/user-attachments/assets/aa336672-17d0-4952-a218-a77901b4d065)
![image](https://github.com/user-attachments/assets/09549a30-f572-4b9f-848b-0566219568c1)

Where do I make a mistake?

I think my question is a kind of basic one but I couldn't find a sort off "Getting Started" document...

**Comment by 5ila5** [comment]
As for the Getting Started document: there is [this](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/installation.md#configuring-sensors), but it is mainly focused on yaml configuration and does not describe exactly how to set this up using the GUI, but the configuration variables match 1:1 the one in the GUI.

It looks like you put `paper` into the types filed, but the Calendar shows `Altpapier` the types field describes which entries to use to calculate the value, as there is no collection entry named `paper` it becomes unknown. If you leave the types field empty, it will match all collection entries.
